viz/charts/linkerd-viz/templates/namespace-metadata.yaml

```diff
@@ -45,7 +45,9 @@ spec:
           runAsUser: {{.Values.defaultUID}}
           seccompProfile:
             type: RuntimeDefault
-        args:
+        - args:
```

Comment:
This doesn't look right to me.  I think you put the security context in the `args` by mistake.

Reply:
Yeah, I indented it a bit too much. Gonna fix it up

Reply:
I don't think this is just an issue with the indentation.  The `args` field has been separated from its contents.  Please check the yml carefully.